js/src/util/validation.js

```diff
@@ -42,9 +44,14 @@ export function validateAbi (abi) {
   try {
     abiParsed = JSON.parse(abi);
 
-    if (!util.isArray(abiParsed)) {
+    if (!apiutil.isArray(abiParsed)) {
```

Comment:
Not an ideological war  - there is a reason the api utilities has the isFunction, isArray, isObject, etc. This is to allow consistency instead of things being done differently all-over. So if the underlying implementations can be made better and filter through to everything, we should do so by all means.